.github/workflows/ci.generate.ts

```diff
@@ -105,6 +105,11 @@ echo "sysroot env:"
 cat /sysroot/.env
 . /sysroot/.env
 
+GCC_DIR=\`find /usr/lib -name "crtbeginS.o" -print -quit | xargs dirname\`
+echo "GCC_DIR=$GCC_DIR"
+
+sudo cp -R $GCC_DIR/** /sysroot/usr/lib/
```

Comment:
does symlink not work?

Reply:
found a better fix: https://github.com/denoland/deno_sysroot_build/pull/2